synthtool/gcp/templates/python_library/.flake8

```diff
@@ -1,4 +1,5 @@
 [flake8]
+ignore = E203, E266, E501, W503
```

Comment:
Ugh, I know I'm late to the party, but it would be way better if this line used named constants. (checks) Double ugh!! `pycodstyle` doesn't *have* named constants!

Could we add a comment there explaining the codes?